tests_e2e/pipeline/scripts/execute_tests.sh

```diff
@@ -70,29 +79,33 @@ docker run --rm \
 || echo "exit $?" > /tmp/exit.sh
 
 #
-# Retake ownership of the source and staging directories (note that the former does not need to be done recursively; also, we don't need to
-# retake ownership of the ssh directory)
+# Re-take ownership of the logs directory
 #
-sudo chown "$USER" "$BUILD_SOURCESDIRECTORY"
-sudo find "$BUILD_ARTIFACTSTAGINGDIRECTORY" -exec chown "$USER" {} \;
+sudo find "$LOGS_DIRECTORY" -exec chown "$USER" {} \;
 
-# The LISA run will produce a tree similar to
 #
-#    $BUILD_ARTIFACTSTAGINGDIRECTORY/lisa/20221130
-#    $BUILD_ARTIFACTSTAGINGDIRECTORY/lisa/20221130/20221130-160013-749
-#    $BUILD_ARTIFACTSTAGINGDIRECTORY/lisa/20221130/20221130-160013-749/environments
-#    $BUILD_ARTIFACTSTAGINGDIRECTORY/lisa/20221130/20221130-160013-749/lisa-20221130-160013-749.log
-#    $BUILD_ARTIFACTSTAGINGDIRECTORY/lisa/20221130/20221130-160013-749/lisa.junit.xml
-#    etc
+# Move the relevant logs to the staging directory
 #
-# Remove the 2 levels of the tree that indicate the time of the test run to make navigation
-# in the Azure Pipelines UI easier. Also, move the lisa log one level up and remove some of
-# the logs that are not needed
-#
-mv "$BUILD_ARTIFACTSTAGINGDIRECTORY"/lisa/[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]/*/* "$BUILD_ARTIFACTSTAGINGDIRECTORY"/lisa
-rm -r "$BUILD_ARTIFACTSTAGINGDIRECTORY"/lisa/[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]
-mv "$BUILD_ARTIFACTSTAGINGDIRECTORY"/lisa/lisa-*.log "$BUILD_ARTIFACTSTAGINGDIRECTORY"
-rm "$BUILD_ARTIFACTSTAGINGDIRECTORY"/lisa/messages.log
+if ls "$LOGS_DIRECTORY"/env-*.log > /dev/null 2>&1; then
```

Comment:
separated the logs into subdirectories:
* root directory: logs for tests with errors
* test_logs: logs for passing tests
* environment_logs: logs for environment (test VM) setup
* vm_logs: logs harvested from the VMs
* runbook_logs: logs global to the runbook (LISA log and junit results)